README.md

```diff
@@ -18,9 +18,9 @@ RE(sim_rel_scan_plan())
 Set up the development environment.
 
 ```bash
-conda create -y -n model_instrument python pyqt=5 pyepics
+conda create -y -n model_instrument python=3.10 pyepics
```

Comment:
Technically, why is required to pin Python here?

Reply:
I don't think it necessarily is

Reply:
Remove the pin.

Reply:
Tested without lock, when not locked now environment fails to build pyarrow:
```
Failed to build pyarrow
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (pyarrow)
```
Tested 3.11. It works. Leave this as is for now. We can test newer versions later